src/java/org/apache/cassandra/cql3/statements/schema/AlterKeyspaceStatement.java

```diff
@@ -99,6 +102,15 @@ public void authorize(ClientState client)
         client.ensureKeyspacePermission(keyspaceName, Permission.ALTER);
     }
 
+    @Override
+    public void validate(ClientState state)
+    {
+        super.validate(state);
```

Comment:
if you rebase I believe this was done by Josh, so you don't need to store the state